docs/pi_imaging.md

```diff
@@ -43,6 +43,15 @@ sudo dd if=/dev/sdaX bs=4MiB | gzip > amplipi.img.gz
 To save time copying, shrink the root partition so that unused space isn't copied.
 While the Pi is mounted with `rpiboot`, use the partition manager of your choice to shrink the root partition as far as possible.
 GParted is a good graphical partitioning tool.
+
+This can also be accomplished through CLI tools:
+```sh
+sudo e2fsck -f /dev/sdaX
+sudo resize2fs -Mp /dev/sdaX # this command will tell you how many blocks it has been shrunk into
+sudo fdisk  /dev/sdaX
+```
+Follow [this guide](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/storage_administration_guide/s2-disk-storage-parted-resize-part), using sector sizes for the end of the partition. Some math is necessary to determine the correct end of the partition using this method. `resize2fs` will print 4k blocks, but `fdisk` provides 512byte sectors.
```

Comment:
This seems like a valid addition to these notes, but again see [image_shrink.bash](https://github.com/micro-nova/AmpliPi/blob/3b903b51d4b1e1cdd91414e17aaab2f964e617c2/scripts/image_shrink.bash#L80) for the automated method.